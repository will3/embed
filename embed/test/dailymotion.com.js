const test = require('./utils/test');

test('dailymotion',
  'http://www.dailymotion.com/video/x5lvnhf_nick-puts-pressure-on-theresa-may-over-if-she-ll-put-taxes-up_news');
