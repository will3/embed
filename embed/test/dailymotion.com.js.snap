// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` dailymotion 1`] = `
Object {
  "description": undefined,
  "originalUrl": "http://www.dailymotion.com/video/x5lvnhf_nick-puts-pressure-on-theresa-may-over-if-she-ll-put-taxes-up_news",
  "site_name": "Dailymotion",
  "thumbnail": Object {
    "url": "https://s1-ssl.dmcdn.net/jVtBs/526x297-WF7.jpg",
  },
  "title": "Nick Puts Pressure On Theresa May Over If She&#039;ll Put Taxes Up - Video Dailymotion",
  "video": Object {
    "description": undefined,
    "height": "360",
    "title": undefined,
    "url": "https://www.dailymotion.com/embed/video/x5lvnhf",
    "width": "640",
  },
}
`;
